torch/utils/benchmark/examples/callgrind.py

```diff
@@ -0,0 +1,36 @@
+"""Demonstrate Callgrind collection in both C++ and Python."""
```

Comment:
Is this file exercised by CI?

Reply:
It isn't. The benchmark utils examples are in a transitional state, serving as pseudo-documentation while the real documentation is being created.